app/routes/entries/browse/$initialLetters/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { type LoaderArgs, redirect } from "@remix-run/server-runtime";
+
+export async function loader({ request, params }: LoaderArgs) {
+    return redirect(`${params.initialLetters}/1`);
+}
+
+export default function SearchIndexPage() {
```

Comment:
I only just realized this myself yesterday (as I am still very much learning Remix!), but if we return a redirect in the `loader` we do not actually need to export a default component function. So everything after the loader can be deleted.

Reply:
Done